packages/fxa-payments-server/src/routes/Product/SubscriptionSuccess/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useContext } from 'react';
+import { Plan, Customer, Profile } from '../../../store/types';
+import { AppContext } from '../../../lib/AppContext';
+
+import { metadataFromPlan } from '../../../store/utils';
+
+import './index.scss';
+import PlanDetails from '../../../components/PlanDetails';
+import PaymentConfirmation from '../../../components/PaymentConfirmation';
+import Header from '../../../components/Header';
+
+const defaultProductRedirectURL = 'https://mozilla.org';
+
+export type SubscriptionSuccessProps = {
+  customer: Customer;
+  plan: Plan;
+  profile: Profile;
+};
+
+export const SubscriptionSuccess = ({
+  customer,
+  plan,
+  profile,
+}: SubscriptionSuccessProps) => {
+  const { product_id } = plan;
+  const { downloadURL } = metadataFromPlan(plan);
+  const {
+    config: { productRedirectURLs },
+    matchMedia,
+  } = useContext(AppContext);
+
+  const isMobile = matchMedia('(min-width: 768px)');
```

Comment:
We still need a resize event here to update the state when going from one to the other. I left a link here that may be helpful: https://github.com/mozilla/fxa/pull/4423#discussion_r388035310